extensions/wikia/RobotsTxt/tests/RobotsTxtTest.php

```diff
@@ -90,10 +90,10 @@ public function dataProviderClassApi() {
 					'User-agent: *',
 					'Allow: /abc',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Allow: /abc" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)